bin/eg-generator.js

```diff
@@ -71,10 +71,10 @@ module.exports = class EgGenerator extends Generator {
       baseURL = systemConfig.cli.url;
     } else if (gatewayConfig && gatewayConfig.admin) {
       const adminConfig = gatewayConfig.admin;
-      const hostname = adminConfig.hostname || 'localhost';
+      const host = adminConfig.host || 'localhost';
```

Comment:
You need also to try to grab the property from the old property that we're not going to deprecate.
```suggestion
      const host = adminConfig.host || adminConfig.hostname || 'localhost';
```

If would be great if you could also delete the default value `localhost` and put it in the default value of the JSON Schema file.